crosswalks/SPDX_2_3.csv

```diff
@@ -0,0 +1,74 @@
+Property,SPDX 2.3
+codeRepository,PackageSourceInfo
+programmingLanguage,
+review,Annotator
+runtimePlatform,
+targetProduct,
+applicationCategory,PrimaryPackagePurpose
+applicationSubCategory,
+downloadUrl,PackageDownloadLocation
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,PackageVersion
+storageRequirements,
+supportingData,
+author,PackageOriginator
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,BuiltDate
+dateModified,
+datePublished,ReleaseDate
+editor,
+encoding,
+fileFormat,FileType
+funder,
+keywords,
+license,PackageLicenseConcluded
+producer,PackageOriginator
+provider,PackageSupplier
+publisher,
+sponsor,
+version,PackageVersion
+isAccessibleForFree,
+isPartOf,CONTAINED_BY
+hasPart,CONTAINS
+position,
+description,PackageSummary / PackageDescription
+identifier,
+name,PackageName
+sameAs,
+url,PackageHomePage
+relatedLink,
+givenName,
+familyName,
+email,
+affiliation,
+identifier,
+name,PackageOriginator
+address,
+reviewAspect,AnnotationType
+reviewBody,Annotator
+endDate,
+roleName,
+startDate,
+softwareSuggestions,
+maintainer,
+continuousIntegration,
+buildInstructions,
+developmentStatus,
+embargoEndDate,
+funding,PackageAttributionText

```

Comment:
This doesn't seem like a very good match as `PackageAttributionText` is for free-form acknowledgement, but I don't see a better option. Should we keep it?

Reply:
May be it's safer to remove it.

Reply:
I have removed it.